App/StackExchange.DataExplorer/Web.config

```diff
@@ -105,111 +105,109 @@
     <rewrite>
       <rules>
         <rule name="odata">
-          <match url="^([^/]+)/atom/?(.*)$" />
-          <action type="Rewrite" url="odata.svc/{R:2}?5D6DA575E16342AEB6AF9177FF673569={R:1}" appendQueryString="true" />
+          <match url="^([^/]+)/atom/?(.*)$"/>
+          <action type="Rewrite" url="odata.svc/{R:2}?5D6DA575E16342AEB6AF9177FF673569={R:1}" appendQueryString="true"/>
         </rule>
         <rule name="cloudexchange" stopProcessing="true">
-          <match url=".*" />
+          <match url=".*"/>
           <conditions>
-            <add input="{HTTP_HOST}" pattern="odata.stackexchange.com" />
+            <add input="{HTTP_HOST}" pattern="odata.stackexchange.com"/>
           </conditions>
-          <action type="Redirect" url="http://data.stackexchange.com/{R:0}" />
+          <action type="Redirect" url="http://data.stackexchange.com/{R:0}"/>
         </rule>
       </rules>
     </rewrite>
-    <urlCompression doDynamicCompression="true" dynamicCompressionBeforeCache="true" />
+    <urlCompression doDynamicCompression="true" dynamicCompressionBeforeCache="true"/>
     <httpCompression noCompressionForHttp10="false" noCompressionForProxies="false" dynamicCompressionDisableCpuUsage="93" dynamicCompressionEnableCpuUsage="93" staticCompressionDisableCpuUsage="99" staticCompressionEnableCpuUsage="99">
-      <scheme name="gzip" dll="%Windir%\system32\inetsrv\gzip.dll" staticCompressionLevel="9" dynamicCompressionLevel="4" />
+      <scheme name="gzip" dll="%Windir%\system32\inetsrv\gzip.dll" staticCompressionLevel="9" dynamicCompressionLevel="4"/>
     </httpCompression>
     <httpErrors errorMode="Custom">
-      <remove statusCode="404" />
-      <error statusCode="404" responseMode="ExecuteURL" path="/404" />
+      <remove statusCode="404"/>
+      <error statusCode="404" responseMode="ExecuteURL" path="/404"/>
     </httpErrors>
-    <modules runAllManagedModulesForAllRequests="true" />
+	  <modules runAllManagedModulesForAllRequests="true">
+		  <add name="ErrorStore" type="StackExchange.Exceptional.ExceptionalModule, StackExchange.Exceptional" />
+	  </modules>

```

Comment:
Nit: tab/space and formatting - suggest doc format so it doesn't render in the next diff :)